gate-web/config/gate.yml

```diff
@@ -81,13 +81,35 @@ spring:
       SORT_PROPERTIES_ALPHABETICALLY: true
     serialization:
       ORDER_MAP_ENTRIES_BY_KEYS: true
-#  oauth2:
-#      client:
-#        clientId:
-#        clientSecret:
-#        accessTokenUri:
-#        userAuthorizationUri:
-#        scope:
-#          - email
-#      resource:
-#        userInfoUri:
+
+---
+
+spring:
+  profiles: googleOAuth
+  oauth2:
+    client:
+      # Override these values in your own gate-googleOAuth.yml file
```

Comment:
overrides can be in their spinnaker-local.yml file (probably better guidance since that is what we generally are telling people to edit)